requirements/base.txt

```diff
@@ -1,21 +1,11 @@
-albumentations==1.1.0
-attrdict==2.0.1
-defusedxml==0.7.1
-einops==0.3.2
-kornia==0.5.6
-lxml==4.6.5
-matplotlib==3.4.3
-networkx~=2.5
-numpy~=1.19.5
-omegaconf==2.1.1
+albumentations>=1.1.0
+einops>=0.3.2
+kornia>=0.5.6
+omegaconf>=2.1.1
 opencv-python>=4.5.3.56
-opencv-contrib-python==4.5.5.62
-pandas~=1.1.5
-pillow==9.0.0
+pandas>=1.1.0
 pytorch-lightning>=1.6.0
-torch==1.8.1
-torchvision==0.9.1
-torchtext==0.9.1
-scikit-image>=0.17.2
-scikit-learn==0.24.2
-wandb==0.12.9
+torchvision>=0.9.1
+torchtext>=0.9.1
+wandb>=0.12.9
```

Comment:
@dreaquil, this also needs to be reverted back to `wandb==0.12.9` for now. For some reason, CI fails due to this. We could address it later.